foundationsync/indico_foundationsync/plugin.py

```diff
@@ -43,11 +51,34 @@ class FoundationSyncPlugin(IndicoPlugin):
     def init(self):
         super(FoundationSyncPlugin, self).init()
         self.connect(signals.plugin.cli, self._extend_indico_cli)
+        self.connect(signals.plugin.schema_post_load, self._check_forbidden_fields, sender=RoomUpdateArgsSchema)
 
     def get_blueprints(self):
         from indico_foundationsync.blueprint import blueprint
         return blueprint
 
+    def _check_forbidden_fields(self, sender, data, **kwargs):
+        """Check that no one is trying to edit fields that come from Gesloc."""
+        sync_collisions = set(data) & SYNCED_FIELDS
+        blocked_collisions = set(data) & BLOCKED_FIELDS
+        messages = {}
+
+        if sync_collisions:
```

Comment:
I guess this `if` is not needed at all.

```suggestion
            messages.update({
                k: [_('This field is managed by Gesloc. You cannot change it from here.')] for k in sync_collisions
            })

```